randovania/games/prime2/json_data/Torvus Bog.txt

```diff
@@ -303,6 +303,9 @@ Extra - in_dark_aether: False
                   Bomb Space Jump (Expert)
                   Movement (Intermediate) and Use Screw Attack (No Space Jump)
                   Gravity Boost and Air Underwater (Advanced)
+                  All of the following:
+                      # https://www.youtube.com/watch?v=LxH7aq404so
+                      Jump Off Enemy (Advanced)
```

Comment:
I would add some amount of damage to this path just in case.

Reply:
I have no idea how this works. From my testing it was easily doable without Energy Tanks. I've set it to >99 for now.

Reply:
If it's 99 or more, then lenient will always require it.